com.unity.render-pipelines.core/Runtime/Textures/PowerOfTwoTextureAtlas.cs

```diff
@@ -233,42 +233,40 @@ public override bool AllocateTexture(CommandBuffer cmd, ref Vector4 scaleOffset,
         public void ResetRequestedTexture() => m_RequestedTextures.Clear();
 
         /// <summary>
-        /// Reserve space from atlas for a texture.
+        /// Reserves the space on the texture atlas
         /// </summary>
-        /// <param name="texture">Source texture.</param>
-        /// <returns>True on success, false otherwise.</returns>
+        /// <param name="texture">The source texture</param>
+        /// <returns>True if the space is reserved</returns>
         public bool ReserveSpace(Texture texture) => ReserveSpace(texture, texture.width, texture.height);
 
         /// <summary>
-        /// Reserve space from atlas for a texture.
+        /// Reserves the space on the texture atlas
```

Comment:
"Reserves the space on the texture atlas" is not grammatically correct IMO, it should be "Reserves space from the texture atlas"

Reply:
```suggestion
        /// Reserves space from the texture atlas
```